src/rawchannel.cc

```diff
@@ -107,7 +110,8 @@ class RawChannel : public Nan::ObjectWrap
   explicit RawChannel(const char *name, bool timestamps, int protocol, bool non_block_send)
     : m_Thread(0), m_Name(name), m_SocketFd(-1)
   {
-    m_SocketFd = socket(PF_CAN, SOCK_RAW, protocol);
+    static const int canfd_on = 1;
```

Comment:
what's the purpose of a member static variable in this case? You are using this variable just once to set the setsockopt for CANFD. const int is sufficient.

Reply:
done